docs/en-US/component/select-v2.md

```diff
@@ -141,38 +141,39 @@ select-v2/use-valueKey
+| disabled                                         | is disabled                                                                                                                              | boolean                            | —                                                                                                         | false         |
+| value-key                                        | unique identity key name for value, required when value is an object                                                                     | string                             | —                                                                                                         | value         |
+| size                                             | input box size                                                                                                                           | string                             | large/default/small                                                                                       | default       |
+| clearable                                        | whether select can be cleared                                                                                                            | boolean                            | —                                                                                                         | false         |
+| clear-icon                                       | custom clear icon                                                                                                                        | `string \| Component`              | —                                                                                                         | CircleClose   |
+| collapse-tags                                    | whether to collapse tags to a text when multiple selecting                                                                               | boolean                            | —                                                                                                         | false         |
+| collapse-tags-tooltip                            | whether show all selected tags when mouse hover text of collapse-tags. To use this, `collapse-tags` must be true                         | boolean                            | true / false                                                                                              | false         |
+| max-collapse-tags<VersionTag version="2.2.30" /> | The max tags number to be shown. To use this, `collapse-tags` must be true                                                               | number                             | —                                                                                                         | 1             |
```

Comment:
Same here: https://github.com/element-plus/element-plus/pull/11378#discussion_r1106035787, once this is done, we can merge this.

Reply:
Done. :D